packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -185,6 +185,13 @@ const TurnGenTasks: GarboTask[] = [
     },
     spendsTurn: false,
   },
+  {
+    name: "Law of Averages",
+    ready: () => myAdventures() <= Math.min(1 + globalOptions.saveTurns, 199),
```

Comment:
All these turn gen tasks actually wait until the last minute, so I just used their code, adjusted for the fact that we never want to use this with > 200 turns.